Code/Legacy/CryCommon/crycommon_files.cmake

```diff
@@ -28,7 +28,7 @@ set(FILES
     IValidator.h
     IWindowMessageHandler.h
     IXml.h
-    INavigationSystem.h
+    MicrophoneBus.h
```

Comment:
I am guessing the MicrophoeBus.h header was missing from the list of CryCommon headers?

Reply:
There was a merge conflict when I merged from development and I messed up resolving it. I've just fixed it.